core/src/main/res/layout/ayp_default_player_ui.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/default_native_ui_layout"
```

Comment:
Let's call this `ayp_default_native_ui_layout`. It's good to use the prefix because when users use the library they also have access to all this resources, the prefix is an easy way to explicitly say that it belongs to the lib.